src/runtime.cc

```diff
@@ -147,16 +187,26 @@ namespace detail {
 		}
 #endif
 
+		if(!s_test_mode && m_cfg->get_tracy_mode() != tracy_mode::off) {
+			if constexpr(CELERITY_TRACY_SUPPORT) {
+				CELERITY_WARN("Profiling with Tracy is enabled. Performance may be negatively impacted.");
+			} else {
+				CELERITY_WARN("CELERITY_TRACY is set, but Celerity was compiled without Tracy support. Ignoring.");
+			}
+		}
```

Comment:
Arguably, this should be in the environment variable validator.

Reply:
I disagree. Warning 1) should not be in `config.cc` because potential slowness is not a concern of the config parser. I had thought about moving 2) in there before, but for consistency I should then make `config::get_tracy_mode()` etc. conditionally available, which adds a lot of preprocessor lines for little gain.